### ISSUE TITLE
Name change?

### ISSUE THREAD

**Issue by tpoisot** [apcshields/autocomplete-bibtex#46]
Since #43 adds support for citeproc, wouldn't it make sense to change the name to *autocomplete-references* or *autocomplete-bib* ?

**Comment by apcshields** [comment]
Sure. Any idea of a smooth way to do that in the Atom package manager?



—

**Comment by tpoisot** [comment]
Yes, it's remarkably easy (apm republish, or something to that effect). First we should (1) decide on a name and (2) merge #43 (maybe #42 too) and resolve the merge conflict.

**Comment by apcshields** [comment]
Great!




I'm working on it....



—

**Comment by tpoisot** [comment]
I solved the merge conflict, and the code for citeproc JSON + YAML is now in master. The command to rename is:

    apm publish --rename new-package-name

I think autocomplete-bibliography is fine (more specific than autocomplete-references)
